x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/index/store/cache/CachedBlobContainerIndexInputTests.java

```diff
@@ -114,7 +117,9 @@ public void testRandomReads() throws Exception {
                     )
                 ) {
                     RecoveryState recoveryState = createRecoveryState();
-                    final boolean loaded = directory.loadSnapshot(recoveryState);
+                    final PlainActionFuture<Void> future = PlainActionFuture.newFuture();
+                    final boolean loaded = directory.loadSnapshot(recoveryState, future);
+                    future.get();
```

Comment:
Maybe only do this line randomly to avoid waiting for prewarming before doing all the reads below?

Reply:
sure done :)